fem/lininteg.hpp

```diff
@@ -35,6 +35,10 @@ class LinearFormIntegrator
    virtual void AssembleRHSElementVect(const FiniteElement &el,
                                        FaceElementTransformations &Tr,
                                        Vector &elvect);
+   virtual void AssembleRHSElementVect(const FiniteElement &el1,
+                                       const FiniteElement &el2,
+                                       FaceElementTransformations &Tr,
+                                       Vector &elvect);
```

Comment:
This addition causes the following warnings from `nvcc`:
```console
.../fem/lininteg.hpp(154): warning: overloaded virtual function "mfem::LinearFormIntegrator::AssembleRHSElementVect" is only partially overridden in class "mfem::BoundaryLFIntegrator"

.../fem/lininteg.hpp(403): warning: overloaded virtual function "mfem::LinearFormIntegrator::AssembleRHSElementVect" is only partially overridden in class "mfem::BoundaryFlowIntegrator"

.../fem/lininteg.hpp(440): warning: overloaded virtual function "mfem::LinearFormIntegrator::AssembleRHSElementVect" is only partially overridden in class "mfem::DGDirichletLFIntegrator"

.../fem/lininteg.hpp(480): warning: overloaded virtual function "mfem::LinearFormIntegrator::AssembleRHSElementVect" is only partially overridden in class "mfem::DGElasticityDirichletLFIntegrator"

```

Reply:
@kmittal2, can you address this?

It would be nice if we can catch these things automatically.